streams/ar.m3u

```diff
@@ -299,11 +299,11 @@ https://ythls.onrender.com/channel/UCiaePeoCqpU8hBHiNrgkzrA.m3u8
 http://186.0.233.76:1935/Garage/smil:garage.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="ElHistonautaTV.ar",El Histonauta TV (360p)
 https://stmv1.cnarlatam.com/elhistonauta/elhistonauta/playlist.m3u8
-#EXTINF:-1 tvg-id="ElNueve.ar",El Nueve (1080p)
-https://octubre-live.cdn.vustreams.com/live/channel09/live.isml/live.m3u8
+#EXTINF:-1 tvg-id="ElNueve.ar",El Nueve (1080p) [Not 24/7]
+https://ythls.onrender.com/channel/UC6Qf8b2jcHHilkcr2wpQ9jA.m3u8
```

Comment:
I don't really recommend removing this one when it's still working. Especially due to the fact that ythls runs on the free edition of Render, so it could cut at the end. Keep both links in instance.